crates/polars-pipe/src/executors/sinks/group_by/aggregates/interface.rs

```diff
@@ -54,6 +54,7 @@ pub(crate) enum AggregateFunction {
     SumU64(SumAgg<u64>),
     SumI32(SumAgg<i32>),
     SumI64(SumAgg<i64>),
+    MeanDate(MeanAgg<i64>),
```

Comment:
Can we remove this for now. We are redesigning the streaming engine and don't want to add stuff atm.

Reply:
Yep, it all felt ugly to me anyway.